<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderMenu</class>
 <widget class="QMainWindow" name="OrderMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OrderMenu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 230);
background-image: url(:/background/bg.jpg);</string>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QGroupBox {
    background-color: #f0f9fc;  /* 淡化的背景色 */
    border: 2px solid #6ba7d9;  /* 更淡的蓝色 */
    border-radius: 10px;
    padding: 2px;
    font-size: 14px;
    font-weight: bold;
    color: #2a4a78;  /* 深蓝色，提高对比度 */
}
QGroupBox::title {
                            subcontrol-origin: margin;
                            subcontrol-position: top center;
                            padding: 0 10px;
                            font-size: 20px;
                            font-weight: bold;
                            color: #005691;
                            }
                          </string>
            </property>
            <property name="title">
             <string>类别</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QCheckBox" name="checkBox_all">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QCheckBox {
    font-size: 16px;
    color: #eeeeee; 
    padding: 3px;
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
    border: 2px solid #6ba7d9;  /* 更淡的蓝色 */
    border-radius: 5px;
    background-color: white;
}

QCheckBox::indicator:checked {
    background-color: #6ba7d9;  /* 更淡的蓝色 */
    border-color: #6ba7d9;  /* 更淡的蓝色 */
}

QCheckBox::indicator:unchecked {
    background-color: white;
    border-color: #6ba7d9;  /* 更淡的蓝色 */
}

QCheckBox:hover::indicator {
    border-color: #8ab9e5;  /* 更淡的蓝色 */
}

QCheckBox:pressed::indicator {
    background-color: #4b8cc3;  /* 更淡的蓝色 */
    border-color: #4b8cc3;  /* 更淡的蓝色 */
}

QGroupBox QCheckBox::indicator {
    margin-left: 5px;
}
</string>
               </property>
               <property name="text">
                <string>所有</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="tristate">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QGroupBox {
    background-color: #f0f9fc;  /* 淡化的背景色 */
    border: 2px solid #6ba7d9;  /* 更淡的蓝色 */
    border-radius: 10px;
    padding: 2px;
    font-size: 14px;
    font-weight: bold;
    color: #2a4a78;  /* 深蓝色，提高对比度 */
}

QCheckBox {
    font-size: 16px;
    color: #2a4a78;  /* 深蓝色，提高对比度 */
    padding: 3px;
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
    border: 2px solid #6ba7d9;  /* 更淡的蓝色 */
    border-radius: 5px;
    background-color: white;
}

QCheckBox::indicator:checked {
    background-color: #6ba7d9;  /* 更淡的蓝色 */
    border-color: #6ba7d9;  /* 更淡的蓝色 */
}

QCheckBox::indicator:unchecked {
    background-color: white;
    border-color: #6ba7d9;  /* 更淡的蓝色 */
}

QCheckBox:hover::indicator {
    border-color: #8ab9e5;  /* 更淡的蓝色 */
}

QCheckBox:pressed::indicator {
    background-color: #4b8cc3;  /* 更淡的蓝色 */
    border-color: #4b8cc3;  /* 更淡的蓝色 */
}

QGroupBox QCheckBox::indicator {
    margin-left: 5px;
}
</string>
               </property>
               <property name="text">
                <string>肉食</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_2">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QGroupBox {
    background-color: #f0f9fc;  /* 淡化的背景色 */
    border: 2px solid #6ba7d9;  /* 更淡的蓝色 */
    border-radius: 10px;
    padding: 2px;
    font-size: 14px;
    font-weight: bold;
    color: #2a4a78;  /* 深蓝色，提高对比度 */
}

QCheckBox {
    font-size: 16px;
    color: #2a4a78;  /* 深蓝色，提高对比度 */
    padding: 3px;
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
    border: 2px solid #6ba7d9;  /* 更淡的蓝色 */
    border-radius: 5px;
    background-color: white;
}

QCheckBox::indicator:checked {
    background-color: #6ba7d9;  /* 更淡的蓝色 */
    border-color: #6ba7d9;  /* 更淡的蓝色 */
}

QCheckBox::indicator:unchecked {
    background-color: white;
    border-color: #6ba7d9;  /* 更淡的蓝色 */
}

QCheckBox:hover::indicator {
    border-color: #8ab9e5;  /* 更淡的蓝色 */
}

QCheckBox:pressed::indicator {
    background-color: #4b8cc3;  /* 更淡的蓝色 */
    border-color: #4b8cc3;  /* 更淡的蓝色 */
}

QGroupBox QCheckBox::indicator {
    margin-left: 5px;
}
</string>
               </property>
               <property name="text">
                <string>蔬菜</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_3">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QGroupBox {
    background-color: #f0f9fc;  /* 淡化的背景色 */
    border: 2px solid #6ba7d9;  /* 更淡的蓝色 */
    border-radius: 10px;
    padding: 2px;
    font-size: 14px;
    font-weight: bold;
    color: #2a4a78;  /* 深蓝色，提高对比度 */
}

QCheckBox {
    font-size: 16px;
    color: #2a4a78;  /* 深蓝色，提高对比度 */
    padding: 3px;
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
    border: 2px solid #6ba7d9;  /* 更淡的蓝色 */
    border-radius: 5px;
    background-color: white;
}

QCheckBox::indicator:checked {
    background-color: #6ba7d9;  /* 更淡的蓝色 */
    border-color: #6ba7d9;  /* 更淡的蓝色 */
}

QCheckBox::indicator:unchecked {
    background-color: white;
    border-color: #6ba7d9;  /* 更淡的蓝色 */
}

QCheckBox:hover::indicator {
    border-color: #8ab9e5;  /* 更淡的蓝色 */
}

QCheckBox:pressed::indicator {
    background-color: #4b8cc3;  /* 更淡的蓝色 */
    border-color: #4b8cc3;  /* 更淡的蓝色 */
}

QGroupBox QCheckBox::indicator {
    margin-left: 5px;
}
</string>
               </property>
               <property name="text">
                <string>水产</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_4">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QGroupBox {
    background-color: #f0f9fc;  /* 淡化的背景色 */
    border: 2px solid #6ba7d9;  /* 更淡的蓝色 */
    border-radius: 10px;
    padding: 2px;
    font-size: 14px;
    font-weight: bold;
    color: #2a4a78;  /* 深蓝色，提高对比度 */
}

QCheckBox {
    font-size: 16px;
    color: #2a4a78;  /* 深蓝色，提高对比度 */
    padding: 3px;
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
    border: 2px solid #6ba7d9;  /* 更淡的蓝色 */
    border-radius: 5px;
    background-color: white;
}

QCheckBox::indicator:checked {
    background-color: #6ba7d9;  /* 更淡的蓝色 */
    border-color: #6ba7d9;  /* 更淡的蓝色 */
}

QCheckBox::indicator:unchecked {
    background-color: white;
    border-color: #6ba7d9;  /* 更淡的蓝色 */
}

QCheckBox:hover::indicator {
    border-color: #8ab9e5;  /* 更淡的蓝色 */
}

QCheckBox:pressed::indicator {
    background-color: #4b8cc3;  /* 更淡的蓝色 */
    border-color: #4b8cc3;  /* 更淡的蓝色 */
}

QGroupBox QCheckBox::indicator {
    margin-left: 5px;
}
</string>
               </property>
               <property name="text">
                <string>豆制品</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="tristate">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_5">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QGroupBox {
    background-color: #f0f9fc;  /* 淡化的背景色 */
    border: 2px solid #6ba7d9;  /* 更淡的蓝色 */
    border-radius: 10px;
    padding: 2px;
    font-size: 14px;
    font-weight: bold;
    color: #2a4a78;  /* 深蓝色，提高对比度 */
}

QCheckBox {
    font-size: 16px;
    color: #2a4a78;  /* 深蓝色，提高对比度 */
    padding: 3px;
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
    border: 2px solid #6ba7d9;  /* 更淡的蓝色 */
    border-radius: 5px;
    background-color: white;
}

QCheckBox::indicator:checked {
    background-color: #6ba7d9;  /* 更淡的蓝色 */
    border-color: #6ba7d9;  /* 更淡的蓝色 */
}

QCheckBox::indicator:unchecked {
    background-color: white;
    border-color: #6ba7d9;  /* 更淡的蓝色 */
}

QCheckBox:hover::indicator {
    border-color: #8ab9e5;  /* 更淡的蓝色 */
}

QCheckBox:pressed::indicator {
    background-color: #4b8cc3;  /* 更淡的蓝色 */
    border-color: #4b8cc3;  /* 更淡的蓝色 */
}

QGroupBox QCheckBox::indicator {
    margin-left: 5px;
}
</string>
               </property>
               <property name="text">
                <string>汤</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_6">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QGroupBox {
    background-color: #f0f9fc;  /* 淡化的背景色 */
    border: 2px solid #6ba7d9;  /* 更淡的蓝色 */
    border-radius: 10px;
    padding: 2px;
    font-size: 14px;
    font-weight: bold;
    color: #2a4a78;  /* 深蓝色，提高对比度 */
}

QCheckBox {
    font-size: 16px;
    color: #2a4a78;  /* 深蓝色，提高对比度 */
    padding: 3px;
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
    border: 2px solid #6ba7d9;  /* 更淡的蓝色 */
    border-radius: 5px;
    background-color: white;
}

QCheckBox::indicator:checked {
    background-color: #6ba7d9;  /* 更淡的蓝色 */
    border-color: #6ba7d9;  /* 更淡的蓝色 */
}

QCheckBox::indicator:unchecked {
    background-color: white;
    border-color: #6ba7d9;  /* 更淡的蓝色 */
}

QCheckBox:hover::indicator {
    border-color: #8ab9e5;  /* 更淡的蓝色 */
}

QCheckBox:pressed::indicator {
    background-color: #4b8cc3;  /* 更淡的蓝色 */
    border-color: #4b8cc3;  /* 更淡的蓝色 */
}

QGroupBox QCheckBox::indicator {
    margin-left: 5px;
}
</string>
               </property>
               <property name="text">
                <string>主食</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_7">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QGroupBox {
    background-color: #f0f9fc;  /* 淡化的背景色 */
    border: 2px solid #6ba7d9;  /* 更淡的蓝色 */
    border-radius: 10px;
    padding: 2px;
    font-size: 14px;
    font-weight: bold;
    color: #2a4a78;  /* 深蓝色，提高对比度 */
}

QCheckBox {
    font-size: 16px;
    color: #2a4a78;  /* 深蓝色，提高对比度 */
    padding: 3px;
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
    border: 2px solid #6ba7d9;  /* 更淡的蓝色 */
    border-radius: 5px;
    background-color: white;
}

QCheckBox::indicator:checked {
    background-color: #6ba7d9;  /* 更淡的蓝色 */
    border-color: #6ba7d9;  /* 更淡的蓝色 */
}

QCheckBox::indicator:unchecked {
    background-color: white;
    border-color: #6ba7d9;  /* 更淡的蓝色 */
}

QCheckBox:hover::indicator {
    border-color: #8ab9e5;  /* 更淡的蓝色 */
}

QCheckBox:pressed::indicator {
    background-color: #4b8cc3;  /* 更淡的蓝色 */
    border-color: #4b8cc3;  /* 更淡的蓝色 */
}

QGroupBox QCheckBox::indicator {
    margin-left: 5px;
}
</string>
               </property>
               <property name="text">
                <string>水果</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_8">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QGroupBox {
    background-color: #f0f9fc;  /* 淡化的背景色 */
    border: 2px solid #6ba7d9;  /* 更淡的蓝色 */
    border-radius: 10px;
    padding: 2px;
    font-size: 14px;
    font-weight: bold;
    color: #2a4a78;  /* 深蓝色，提高对比度 */
}

QCheckBox {
    font-size: 16px;
    color: #2a4a78;  /* 深蓝色，提高对比度 */
    padding: 3px;
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
    border: 2px solid #6ba7d9;  /* 更淡的蓝色 */
    border-radius: 5px;
    background-color: white;
}

QCheckBox::indicator:checked {
    background-color: #6ba7d9;  /* 更淡的蓝色 */
    border-color: #6ba7d9;  /* 更淡的蓝色 */
}

QCheckBox::indicator:unchecked {
    background-color: white;
    border-color: #6ba7d9;  /* 更淡的蓝色 */
}

QCheckBox:hover::indicator {
    border-color: #8ab9e5;  /* 更淡的蓝色 */
}

QCheckBox:pressed::indicator {
    background-color: #4b8cc3;  /* 更淡的蓝色 */
    border-color: #4b8cc3;  /* 更淡的蓝色 */
}

QGroupBox QCheckBox::indicator {
    margin-left: 5px;
}
</string>
               </property>
               <property name="text">
                <string>酒水</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pBtn_play">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton { background-color: #4CAF50; color: white; border-radius: 5px; padding: 10px; }
QPushButton:hover { background-color: #45a049; }
QPushButton:pressed { background-color: #388e3c; }</string>
            </property>
            <property name="text">
             <string>去支付</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/icons/paly.png</normaloff>:/icons/paly.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pBtn_shopping">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>购物车</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/icons/shopping.png</normaloff>:/icons/shopping.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
